### PR TITLE
SERVER-82135: Update the FCV constant in the logkeeper snapshot workload for 7.2

### PR DESCRIPTION
**Pull request by miguelangelnieto**
Created the new snapshot with updated FCV.

### CODE REVIEW

File: etc/system_perf.yml [hunk 1]
```diff
@@ -3896,29 +3896,29 @@ buildvariants:
...
-  #    project_dir: *project_dir
-  #    compile_variant: "-arm64"
-  #  run_on:
-  #    - "rhel70-perf-initialsync-logkeeper"
-  #  depends_on: *_compile_amazon_linux2_arm64
-  #  tasks:
-  #    - name: schedule_patch_auto_tasks
-  #    - name: schedule_variant_auto_tasks
-  #    - name: initialsync-logkeeper-snapshot-update
+  # - name: linux-replSet-initialsync-logkeeper-snapshot-update
```
Comment (jimoleary) [inline etc/system_perf.yml:3899]: do we need this whitespace changes?

**Comment by miguelangelnieto** [comment]
Patch test [here](https://spruce.mongodb.com/version/652e6130c9ec441262d98a26/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

File: etc/system_perf.yml [hunk 2]
```diff
@@ -3896,29 +3896,29 @@ buildvariants:
...
+  #   display_name: Linux ReplSet Initial Sync LogKeeper Snapshot Update
+  #   cron: "0 0 * * 4"  # 00:00 on Thursday
+  #   modules: *modules
+  #   expansions:
+  #     mongodb_setup_release: 2022-11
+  #     mongodb_setup: initialsync-logkeeper-snapshot-update
+  #     infrastructure_provisioning_release: 2022-11
+  #     infrastructure_provisioning: initialsync-logkeeper-snapshot-update
+  #     # Update this to latest snapshot after each LTS release.
+  #     snapshotId: snap-063661c31a4e4b812
```
Comment (jimoleary) [inline etc/system_perf.yml:3909]: Shouldn't this be updated to snap-042ca2a003313ebc2 also ?

**Review by jimoleary — COMMENTED** [review]
just some minor questions.

File: etc/system_perf.yml (hunk 2 above)
Comment (miguelangelnieto) [reply etc/system_perf.yml]: Oh, nice catch. Updated.

**Review by jimoleary — APPROVED** [review]
lgtm!

**Pull request merged**